### ISSUE TITLE
Online documentation for scitokens Python package

### ISSUE THREAD

**Issue by duncanmmacleod** [scitokens/scitokens#151]
Is there a plan to revive the online documentation for the scitokens Python package? <https://scitokens.org/scitokens/> has a 0.1 version of the docs, which is now quite out of date.

I would be happy to help configure readthedocs.io integration (which would then serve the docs at <https://scitokens.readthedocs.io>), which would be completely automated.

**Comment by djw8605** [comment]
Yeah, lets go with read the docs.  Please, feel free to add the integration.

**Comment by duncanmmacleod** [comment]
@djw8605, can you please reopen this as a reminder for me and others until the integration is actually configured?

**Comment by duncanmmacleod** [comment]
@djw8605, I think I will need to be made a maintainer of this project in order to configure the integration. If that's too much of a reach I can walk you (or someone else) through the necessary steps.

**Comment by duncanmmacleod** [comment]
I think this is now complete.
